std/range/package.d

```diff
@@ -1849,19 +1849,30 @@ if (isRandomAccessRange!(Unqual!R) && hasLength!(Unqual!R) && hasSlicing!(Unqual
 Lazily takes only up to `n` elements of a range. This is
 particularly useful when using with infinite ranges.
 
-Unlike $(LREF takeExactly), `take` does not require that there
-are `n` or more elements in `r`. As a consequence, length
-information is not applied to the result unless `r` also has
+Unlike `takeExactly`, `take` does not require that there
```

Comment:
what is the motivation for removing LREF? It creates nice, clickable __L__ocal reference, e.g.:

http://dlang.org/phobos/std_range.html#.take